maps/site53/site53-1.dmm

```diff
@@ -16023,7 +16054,9 @@
 /turf/simulated/floor/tiled/white,
 /area/site53/uhcz/scp8containment)
 "Mm" = (
-/obj/machinery/job_terminal,
+/obj/machinery/job_terminal{
+	locked = 0
```

Comment:
does this even matter? they can't be given jobs until a guard mans the post anyways

Reply:
yes.. a guard has to manually unlock the auto assignment machines